src/navigators/RootStack.tsx

```diff
@@ -37,9 +37,7 @@ export function RootStack() {
 
   const loadingProject = !isFetchedProject || isLoadingProject;
   const loadingAccount = !isFetchedAccount || isLoadingAccount;
-  const hasAccount = !loadingAccount && account?.id;
-  const loadingAccountOrProject =
-    loadingAccount || (hasAccount && loadingProject);
+  const loadingAccountOrProject = loadingAccount || loadingProject;
```

Comment:
I think this should be fine now that the useSubjectProjects query doesn't load indefinitely if there is no account.

Reply:
🤔 are we sure?  Make sure to test the heck of a pure "user is not in ANY accounts' groups" scenario.  I'm looking at [this line](https://github.com/lifeomic/react-native-sdk/blob/master/src/hooks/useSubjectProjects.tsx#L37) and thinking `loadingProject` will be true until an account.  That's what the more nuanced line 42 was handling.